plugins/orchestrator-backend/dist-dynamic/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@janus-idp/backstage-plugin-orchestrator-backend-dynamic",
-  "version": "1.6.2",
+  "version": "1.6.3",
```

Comment:
Please revert this change. Versions are controlled by the semantic release process.